### PR TITLE
[php/spiral] Add RoadRunner 2.0 support for Spiral Framework

### PR DESCRIPTION
**Pull request by butschster**
I also updated CycleORM up to 2.0

### CODE REVIEW

**Comment by butschster** [comment]
Hi @joanhey!

I've just fixed problem with running application.

File: frameworks/PHP/spiral/spiral.dockerfile [hunk 1]
```diff
@@ -11,12 +15,12 @@ RUN chmod +x /usr/local/etc/php/install-composer.sh && /usr/local/etc/php/instal
...
 
 # pre-configure
-RUN ./vendor/bin/spiral get > /dev/null 2>&1
+RUN ./vendor/bin/rr get-binary > /dev/null 2>&1
 RUN php app.php configure > /dev/null 2>&1
 
 EXPOSE 8080
 
-CMD php app.php up > /dev/null 2>&1 && ./spiral serve -o "http.workers.pool.numWorkers = 64"
+CMD php app.php up > /dev/null 2>&1 && ./rr serve -o "http.workers.pool.numWorkers = 64"
```
Comment (rustatian) [inline frameworks/PHP/spiral/spiral.dockerfile:26]: @butschster Should be `http.pool.num_workers = 64`

**Comment by joanhey** [comment]
Hi @butschster 
It's failing in the runs in Citrine.
```
spiral: Step 11/13 : RUN php app.php configure
spiral:  ---> Running in e74137c8a375
spiral:  [Spiral\Config\Exception\LoaderException]                     
spiral:  Unable to load config `scaffolder`: no suitable loader found. 
spiral: in /spiral/vendor/spiral/framework/src/Config/src/Loader/DirectoryLoader.php:64

```

Could you fix it ?

**Comment by butschster** [comment]
> Hi @butschster It's failing in the runs in Citrine.
> 
> ```
> spiral: Step 11/13 : RUN php app.php configure
> spiral:  ---> Running in e74137c8a375
> spiral:  [Spiral\Config\Exception\LoaderException]                     
> spiral:  Unable to load config `scaffolder`: no suitable loader found. 
> spiral: in /spiral/vendor/spiral/framework/src/Config/src/Loader/DirectoryLoader.php:64
> ```
> 
> Could you fix it ?

Hi, fixed!

**Pull request merged**